DeliverySystem/src/App.tsx

```diff
@@ -1,35 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { Container, Grid2, Paper } from '@mui/material';
+import { Header } from './Components/Header/Header';
+import { HomePage } from './Components/pages/HomePage/HomePage';
+import { AuthForm } from './Components/AuthForm/AuthForm';
+import { useState } from 'react';
+import { Routes, Route, useLocation} from 'react-router-dom';
+import { CREATE_ACCOUNT_FIELDS, SIGN_IN_FIELDS, INITIAL_VALUES_CREATE_ACCOUNT, INITIAL_VALUES_SIGN_IN } from './Components/AuthForm/constants';
+import { HOME_PAGE_URL, SIGN_IN_URL, CREATE_ACCOUNT_URL, CREATE_ACCOUNT_FORM_TITLE, SIGN_IN_FORM_TITLE } from './constants';
 
 function App() {
-  const [count, setCount] = useState(0)
-
+  const location = useLocation();
+  const [signInAuth, setSignInAuth] = useState<boolean>(location.pathname === SIGN_IN_URL);
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+      <Container maxWidth='xl'>
+        <Header setSignInAuth={setSignInAuth} />
+        <Grid2>
+          <Paper elevation={3}>
+            <Routes>
+              <Route path={HOME_PAGE_URL} element={<HomePage />} />
+              <Route path={SIGN_IN_URL} element={
+                <AuthForm title={SIGN_IN_FORM_TITLE} 
+                textFields={SIGN_IN_FIELDS} 
+                initialValues={INITIAL_VALUES_SIGN_IN} 
+                signInAuth={signInAuth} 
+              />} 
+              />
+              <Route path={CREATE_ACCOUNT_URL} element={
```

Comment:
one thing I would suggest is to keep the app.tsx clean. like what you did for the home page, cretae a sign in page and create account page and replace the element with them. ( could be a component just to keep this file clean )